doc/examples.md

```diff
@@ -263,4 +263,30 @@ Setting the hostname of a system is as simple as writing `/etc/hostname`:
 }
 ```
 
+## Adding users
+
+Users can be added to a CoreOS system with the `passwd.users` key which takes a list of objects that specify a given user. If you wanted to configure a user "core" and a user "jenkins" you would do that as follows:
+
+```json
+"passwd": {
+  "users": [
+    {
+      "name": "core",
```

Comment:
Can you use a different user here? I'd rather not tie "Container Linux"-specifics to these docs (that's the purpose of https://github.com/coreos/docs/pull/1012).